alws/crud/build_node.py

```diff
@@ -709,6 +726,10 @@ async def build_done(db: Session, pulp: PulpClient,
             binary_rpm.build = build_task.build
             binary_rpms.append(binary_rpm)
 
+    # TODO: Temporary solution that fixes integrity error with missing srpm,
```

Comment:
Can you create a JIRA issue and reference it in the comment?

Reply:
Done